architecture/adr-005-messaging.md

```diff
@@ -0,0 +1,54 @@
+# Architecture Decision Record: Messaging
+
+## Context
+
+As [ADR-004](adr-004-module-loading.md) states, ComposeUI needs to be as modular as possible.
+This means any module and functionality should be configurable ([ADR-002](adr-002-configuration.md)).
+One of the core modules of ComposeUI is the messaging module, or the message router. The message router
+should be responsible for every communication that happens between processes.
+It should be an independent pluggable module that runs under the main ComposeUI process, delivering
+messages between processes and computers.
```

Comment:
Should this read:  and optionally other devices?

Reply:
Actually yes, I guess with web-first there could be other devices.